common/build-style/go.sh

```diff
@@ -41,7 +41,7 @@ do_build() {
 }
 
 do_install() {
-	for f in "$(find "${GOPATH}/bin" -type f -executable)"; do
+	for f in $(find "${GOPATH}/bin" -type f -executable); do
```

Comment:
Is it possible/better to avoid find? Something like:
```sh
for f in "${GOPATH}"/* "${GOPATH}"/**/*; do
    if [ -f "$f" ] && [ -x "$f" ]; then
        vbin "$f"
    fi
done
```
-----
I measured with my "/usr/lib", glob version is 3 times faster than find
```sh
% time ./globtest
./globtest  4.89s user 1.51s system 98% cpu 6.468 total
% time ./findtest
./findtest  5.58s user 13.57s system 98% cpu 19.505 total
% cat globtest
#!/bin/bash
XPATH=/usr
for i in {1..100}; do
        for f in "$XPATH"/lib/* "$XPATH"/lib/**/*; do
                if [ -f "$f" ] && [ -x "$f" ]; then
                        :
                fi
        done
done
% cat findtest
#!/bin/bash
XPATH=/usr
for i in {1..100}; do
        for f in $(find "$XPATH/lib" -type f -executable); do
                :
        done
done
```

Reply:
Avoid forking is nice, IMHO.